.gitignore

```diff
@@ -3,3 +3,4 @@ client/dist
 games/
 server/.lake
 **/.DS_Store
+/.lake
```

Comment:
there should be no `lean4game/.lake` folder on your system, only one at `lean4game/server/.lake`